requirements.txt

```diff
@@ -0,0 +1,5 @@
+numpy==1.15.3
```

Comment:
Remove file. Dependency management is done with [poetry](https://python-poetry.org/)